modules/ding_debt_easy/ding_debt_easy.module

```diff
@@ -0,0 +1,908 @@
+      }
+    }
+  }
+}
+
+/**
+ * Access callback with redirect for content pages generated by the module.
+ *
+ * @param string $type
+ *   The type of page.
+ *
+ * @return bool
+ *   True if the page should be accessed.
+ */
+function _ding_debt_easy_pages_access_callback($type) {
+  $config = variable_get('ding_debt_easy_' . $type, []);
+
+  // Redirect to page outside this module if given in configuration.
+  if (!empty($config['url'])) {
+    drupal_goto($config['url']);
+  }
+
+  return TRUE;
+}
+
+/**
+ * The currencies available at the payment gateway.
+ *
+ * Sadly there is no endpoint to fetches these. So it's created based on
+ * https://nets-devs.isotop.se/nets-easy/en-EU/api/#currency-and-amount.
+ *
+ * @return string[]
+ *   Array indexed by short name for currencies.
+ */
+function _ding_debt_easy_get_currency_options() {
+  return [
+    'DKK' => 'Danish krone',
+    'EUR' => 'Euro',
+    'GBP' => 'Pound sterling',
+    'NOK' => 'Norwegian krone',
+    'SEK' => 'Swedish krona',
+    'USD' => 'United States dollar',
+  ];
+}
+
+/**
+ * Get current configured currency (default: DKK).
+ *
+ * @return string
+ *   The short name for the currently configured currency.
+ */
+function ding_debt_easy_get_currency() {
+  $config = variable_get('ding_debt_easy_config', []);
+  return empty($config['currency']) ? 'DKK' : $config['currency'];
+}
+
+/**
+ * The list of language select at the gateway.
+ *
+ * See https://nets-devs.isotop.se/nets-easy/en-EU/api/#localization.
+ *
+ * @return string[]
+ *   List of languages keys by localization id/code.
+ */
+function _ding_debt_easy_get_localization_options() {
+  return [
+    'da-DK' => 'Danish',
+    'en-GB' => 'English',
+    'nl-NL' => 'Dutch',
+    'fi-FI' => 'Finnish',
+    'fr-FR' => 'French',
+    'de-DE' => 'German',
+    'nb-NO' => 'Norwegian',
+    'pl-PL' => 'Polish',
+    'es-ES' => 'Spanish',
+    'sk-SK' => 'Slovak',
+    'sv-SE' => 'Swedish',
+  ];
+}
+
+/**
+ * Return the selected wipe interval in seconds.
+ *
+ * Default: 30 days.
+ *
+ * @return int
+ *   The select wipe interval.
+ */
+function _ding_debt_easy_get_wipe_interval() {
+  $config = variable_get('ding_debt_easy_config', []);
+  return empty($config['wipe_interval']) ? 86400 * 30 : $config['wipe_interval'];
+}
+
+/**
+ * Get current localization id/code.
+ *
+ * @return string
+ *   Current selected localization land id/code.
+ */
+function ding_debt_easy_get_localization() {
+  $config = variable_get('ding_debt_easy_config', []);
+  return empty($config['lang']) ? 'da-DK' : $config['lang'];
+}
+
+/**
+ * Get list of integration types.
+ *
+ * Currently, the embedded type is not implemented, but it's very possible to do so.
+ *
+ * @return string[]
+ *   List of integration types.
+ */
+function _ding_debt_easy_get_integration_type_options() {
+  return [
+    'HostedPaymentPage' => 'Hosted Payment Page',
+    // 'EmbeddedCheckout' => 'Embedded Checkout (not implemented)',
```

Comment:
It's has since this code was made be decided that "EmbeddedCheckout" should never be implemented in the current CMS... So I will not change this to constants for the time being.